app/src/main/java/org/gdg/frisbee/android/eventseries/TaggedEventSeriesFragment.java

```diff
@@ -74,10 +87,27 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        View v = inflater.inflate(R.layout.fragment_events, container, false);
-        ButterKnife.inject(this, v);
-        return v;
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        if (getArguments() != null && getArguments().getBoolean(ARGS_IS_PORTRAIT, false)) {
+            View header = getLayoutInflater(null)
```

Comment:
It won't be a problem becuase it is tied to the Activity and Activity is recreated when a configurationChange occurs. 
Actually it is not just about Portrait, I will change here to make more sense.